index.html

```diff
@@ -15,9 +16,44 @@
   </head>
 
   <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+  <div class="container">
+    <p class="logoOne">Relaxr</p>
+    <p class="message">Get a piece of mind with a single tap.</p>
+    <div class="centerButton">
+      <button class="getIt">Get it now!</button>
+    </div>
+  </div>
+  <section class="greyBackground">
+    <div class="productDescription">
+      <p class="benefits">Benefits</p>
+      <p class="aboutRelaxr">The perfect personal assistant. Relaxr does your job </br> for you so you can enjoy life as 
+       it is meant to be.</p>
+      <ul class="listStyle">
+        <li>Schedule meetings for you</li>
+        <li>Excel automation to complete your</br> work for you</li>
+        <li>Responds to emails on your behalf</li>
+        <li>Does all your work for you with our</br> revolutionary AI technology</li>
+      </ul>
+    </div>
+  </section>
+  <div class="quote">
+    <p class="testimonial">"Relaxr changed my life. I’ve been able</br> to travel the world, spend 
+      limited time</br> working and my boss keeps thanking </br> me for crushing work."</br><strong 
+      class="amanda">-Amanda, Intuit</strong></p>
+  </div>
+  <div class="yellowBackground">
+      <div class="centerButton">
+        <button class="signUp">Sign up now!</button>
+      </div>
+  </div>
+  <section class="darkGreyBackground"> 
```

Comment:
another tag that would have worked well here instead of `section` is the `footer` tag. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer